app/models/publications/monthly_statistics/monthly_statistics_report.rb

```diff
@@ -3,8 +3,8 @@ module MonthlyStatistics
     class MonthlyStatisticsReport < ApplicationRecord
       validates :statistics, :generation_date, :publication_date, :month, presence: true
 
-      scope :published, -> { where('publication_date <= ?', Time.zone.now) }
-      scope :drafts, -> { where('publication_date > ?', Time.zone.now) }
+      scope :published, -> { where('publication_date <= ?', Time.zone.today) }
+      scope :drafts, -> { where('publication_date > ?', Time.zone.today) }
```

Comment:
This is necessary as `Time.zone.today <= Time.zone.now == false`.  Both things have to be dates instead of comparing a date with a date_time.